src/semgrep_agent/main.py

```diff
@@ -162,7 +162,7 @@ def main(
         click.secho(str(error), err=True, fg="red")
         _handle_error(str(error), 2, sapp, meta)
     except InvalidGitRepositoryError as error:
-        click.secho("Current directory is not a github repository", err=True, fg="red")
+        click.secho("Current directory is not a git repository", err=True, fg="red")
```

Comment:
I had this fixup lingering on my local copy of the repo and didn't think it was worth its own PR so thought I would sneak it in here 🤷

Reply:
If any tests fail or anything, we can just revert it - don't want to slow down this PR!